docs/blog/posts/Kubescape-support-for-VEX-generation/kubescape-support-for-vex-generation.md

```diff
@@ -0,0 +1,215 @@
+---
+date: 2023-12-07
+categories:
+  - Announcements
+  - Project
+authors:
+  - oshratnir
```

Comment:
This seems to fail the build:
```
2:50:15 PM: ERROR   -  Error reading page "blog/posts/Kubescape-support-for-VEX-generation/kubescape-support-for-vex-generation.md":
2:50:15 PM: ERROR   -  Couldn"t find author "oshratnir"
2:50:15 PM: Aborted with a BuildError!
```
Can it be that [Oshratn](https://github.com/Oshratn) needs to be used here instead of `oshratnir`?

Reply:
fixed